lib/internal/readline/interface.js

```diff
@@ -78,6 +78,11 @@ const kQuestionCancel = Symbol('kQuestionCancel');
 // GNU readline library - keyseq-timeout is 500ms (default)
 const ESCAPE_CODE_TIMEOUT = 500;
 
+// Store deleted stuff in kill ring
```

Comment:
Would be worthwhile to expand the comment here to explain what the kill ring is.